test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -361,6 +361,12 @@ TEST_F(QuicPlatformTest, QuicLog) {
 #define VALUE_BY_COMPILE_MODE(debug_mode_value, release_mode_value) debug_mode_value
 #endif
 
+TEST_F(QuicPlatformTest, LogEndOfLine) {
```

Comment:
Can we test a few more [io manipulators](https://en.cppreference.com/w/cpp/io/manip)? Some code may use std::hex, std::dec, std::fixed etc.

Also please change the test name to something like QuicLogIoManipulators.

Reply:
done